Examples/MAX32665/Demo/README.md

```diff
@@ -0,0 +1,41 @@
+## Description
```

Comment:
Does the MAX32665 project need a copy of the bmp2c utility as well?

Reply:
Nope, deleted.